torch/_refs/__init__.py

```diff
@@ -1808,6 +1815,75 @@ def narrow(a: TensorLikeType, dim: int, start: int, length: int) -> TensorLikeTy
     return prims.slice_in_dim(a, start, start + length, axis=dim)
 
 
+def _normalize(a, norm_dims, eps):
```

Comment:
Add a comment for this function -- what it's meant to be used for, how it's intended to be used -- we should probably add type annotations to it